src/queries.ts

```diff
@@ -213,75 +213,48 @@ export function appQuery(
   return querystr_to_array(code);
 }
 
+// To see actual app names as discovered by users, see older pre-regex commits.
 const browser_appnames = {
   chrome: [
     // Chrome
-    'Google Chrome',
-    'Google-chrome',
+    /google\w?chrome(-beta|-stable|-unstable)?/i,
     'chrome.exe',
-    'google-chrome-stable',
 
     // Chromium
-    'Chromium',
-    'Chromium-browser',
-    'Chromium-browser-chromium',
-    'chromium.exe',
-
-    // Pre-releases
-    'Google-chrome-beta',
-    'Google-chrome-unstable',
+    /chromium\w?(browser)?(-chromium)?(.exe)?/i,
 
     // Brave (should this be merged with the brave entry?)
     'Brave-browser',
```

Comment:
Just curious — Is there some reason for not moving these lines to the Brave Browser category?

Reply:
I don't remember...

Found https://discord.com/channels/755040852727955476/755334543891759194/1237102962241704038

Looks like we should merge: https://github.com/ActivityWatch/aw-webui/pull/283

But something stalled it.